### DISCUSSION TITLE
Can someone else join my vault?

### DISCUSSION THREAD

**Discussion by WikiJS-pixel** [remotely-save/remotely-save#839] [Q&A]
Hello,

Can someone else join my vault to view and edit my notes? E.g. my colleagues.

**Comment by fyears** [comment]
i don't think the plugin is designed to work in this way.

however, if you have a webdav server and the password is shared across everyone, then everyone can login and the plugin will behave like across lots of devices.

i strongly NOT recommend doing so though
